test/ibmq/test_experiment.py

```diff
@@ -458,6 +460,7 @@ def test_update_analysis_result(self):
                 self.assertEqual(res.type, original_type)
                 self.assertEqual(res.quality, ResultQuality.HUMAN_BAD)
                 self.assertTrue(res.updated_datetime.tzinfo)
+                self.assertTrue(res.verified)
```

Comment:
Given we want to test whether `ibmq-provider` is sending the right data to the server, not whether the server does the correct mapping, I'd suggest adding `new_result.verified = True` after line 450 instead of relying on `HUMAN_BAD` mapping.